chrome/background.js

```diff
@@ -0,0 +1,46 @@
+/*
+Background.html
```

Comment:
If you define scripts in the background section of the manifest, a "background.html" default is created that just loads your script files.